pkg/restapi/operation/operations.go

```diff
@@ -283,6 +301,18 @@ func (c *Operation) readDocumentHandler(rw http.ResponseWriter, req *http.Reques
 	writeReadDocumentSuccess(rw, documentBytes, docID, vaultID)
 }
 
+// createConfigVault creates a store that stores DataVaultConfigurations of data vaults
```

Comment:
There's some redundancy in this comment.

`...stores DataVaultConfigurations of data vaults`

DataVaultConfigurations are, by definition, for data vaults. so you can just simplify this comment to:

`// createConfigVault creates a store that stores DataVaultConfigurations`